auth/src/main/java/org/jboss/pnc/auth/JAASLoggedInUser.java

```diff
@@ -39,7 +39,7 @@ public JAASLoggedInUser(HttpServletRequest httpServletRequest) {
 
     @Override
     public String getEmail() {
-        return "Email N/A (" + getUserName() + ")";
+        return "example@example.com";
```

Comment:
Although this class is currently not used in our production deployment it can be (generally looking, it is not for test only). The previous return string is better fit than hard-coded example.com.
You probably changed it due to email validation constraint. I see two options here:
a) update the class to read the email and other missing data from the User entity, in this case it would mean, if you use JAAS you should have another way to register new users (For tests we can use demo-data).
b) remove the constraint